### ISSUE TITLE
Link to non-existent file needs review

### ISSUE THREAD

**Issue by ileil** [angular/angular#46199]
### Description

The former link for persistent disk cache in workspace-config pointed to a non-existing file. The link should work or be plain text. 

### What is the affected URL?

angular.io/guide/workspace-config

### Please provide the steps to reproduce the issue

| Property              | Details                                                                                       | Value type                                              |
|:---                   |:---                                                                                           |:---                                                     |
| `analytics`           | Share anonymous [usage data](cli/usage-analytics-gathering) with the Angular Team.            | `boolean` &verbar; `ci`                                 |
| `analyticsSharing`    | A set of analytics sharing options.                                                           | [Analytics sharing options](#analytics-sharing-options) |
| `cache`               | Control **persistent disk cache** used by [Angular CLI Builders](guide/cli-builder). | [Cache options](#cache-options)                         |
| `schematicCollections`| A list of default schematics collections to use.                                              | `string[]`                                              |
| `packageManager`      | The preferred package manager tool to use.                                                    | `npm` &verbar; `cnpm` &verbar; `pnpm` &verbar;`yarn`    |
| `warnings`            | Control CLI specific console warnings.                                                        | [Warnings options](#warnings-options)                   |

### Please provide the expected behavior vs the actual behavior you encountered

The link for persistent disk cache should work or be plain text. 

### Please provide a screenshot if possible

n/a

### Please provide the exception or error you saw

```true
n/a
```

### Is this a browser-specific issue? If so, please specify the device, browser, and version.

```true
n/a
```

**Comment by gkalpak** [comment]
The link to https://angular.io/cli/cache work fine for me. What is the link URL that does not work for you?

**Comment by ileil** [comment]
Talked to George Kalpakas. The cache.md file has indeed been removed, but it has been replaced by the cache CLI command (which is automatically generated during the build). So, the deployed site should have a page to show at https://angular.io/cli/cache. Locally I ran the following commands to show the linked /cli/cache page:

1. yarn cache clean
2. yarn setup 
3. yarn start (or yarn serve-and-sync) 
4. Open a Chrome browser window and go to http://localhost:4200/cli/cache.
